sdk/vision/azure-ai-vision-imageanalysis/tests/README.md

```diff
@@ -0,0 +1,53 @@
+# Azure Image Analysis client library tests for Python
+
+## Running tests locally, on a Windows PC, against the live service
+
+### Prerequisites
+
+See [Prerequisites](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/vision/azure-ai-vision-imageanalysis/README.md#prerequisites). Create an Azure resource in one of the GPU-supported regions, otherwise some of the tests will fail.
+
+### Setup
+
```

Comment:
I guess it is not required to build their own wheel to run the tests, right? :)

After we ship the package, user can just install it from pypi.

Reply:
I guess I was only thinking about future developers (me or others...) who will update the SDK and will want to re-test it. But you have a good point @xiangyan99 . I updated this README.md to mention both options: running against latest published client library, or installing local wheel and running tests against it. Thank you!